src/metadataGeneration/resolveType.ts

```diff
@@ -20,7 +20,7 @@ type UsableDeclaration = ts.InterfaceDeclaration
   | ts.ClassDeclaration
   | ts.TypeAliasDeclaration;
 
-export function ResolveType(typeNode: ts.TypeNode): Type {
+export function ResolveType(typeNode: ts.TypeNode, depth = 1): Type {
```

Comment:
ok, I found the doc: http://typedoc.org/api/interfaces/ts.methoddeclaration.html#type